config/middlewares/validations.js

```diff
@@ -0,0 +1,26 @@
+const { check, validationResult } = require('express-validator/check');
+/**
+ * @description - Validate user signup
+ * @param {object} req - request body
+ * @param {object} res - response body
+ *
+ * @return {undefined} depends on user input
+ */
+exports.validateSignup = (req, res, next) => {
+  // Finds the validation errors in this request and wraps them in an object with handy functions
+  const errors = validationResult(req);
+  console.log(errors);
```

Comment:
Avoid leaving `logs` on your code. Please get rid of it @OlatunjiYso

Reply:
Thanks, It has been removed.